chart/values.staging.yaml

```
@@ -7,7 +7,7 @@ ingress:
     traefik.ingress.kubernetes.io/router.tls: "true"
 
 controller:
-  image: moleculemaker/novostoic-frontend:latest
+  image: moleculemaker/novostoic-frontend:pr-34
```

You might want to revert this change before merging :+1:
```suggestion
  image: moleculemaker/novostoic-frontend:latest
```